static/js/formValidation.js

```diff
@@ -134,8 +136,22 @@ afterSubmit = function(download_asset_url, return_url, isModal, thankYouMessage)
     if (thankYouMessage === null) {
       thankYouMessage = 'Thank you<br />A member of our team will be in touch within one working day';
     }
-    document.getElementById('main-content').insertAdjacentHTML('afterbegin', '<div class="p-strip is-shallow u-no-padding--bottom"><div class="row"><div class="p-notification--positive"><p class="p-notification__response"><span class="p-notification__status">Success:</span> ' + thankYouMessage + '</p></div></div></div>');
-    window.scrollTo(0,0);
+    var feedbackArea = document.querySelector('.js-feedback-notification');
+    if (feedbackArea) {
+      feedbackArea.innerHTML = '<div class="p-notification--positive"><p class="p-notification__response">' + thankYouMessage + '</p></div>';
+      var inputs = marketoForm.querySelectorAll('input, button');
+      for (var i = 0; i < inputs.length; i++) {
+        inputs[i].disabled = "disabled";
+      }
+      var recaptcha = marketoForm.querySelector('.g-recaptcha');
+      if (recaptcha) {
+        recaptcha.classList.add('u-hide');
```

Comment:
Is there a reason for hiding the Captcha? It has a drawback — the “Subscribe and download” button jumps upward as soon as you click it.

Reply:
Yea, I didn't want to but as its an iframe I cannot disable it so dont want people playing with it when it looks disabled.

Reply:
Would it work to change the iframe `src` to `about:blank` instead?

I don’t see any other problems, +1